common.yaml

```diff
@@ -0,0 +1,47 @@
+openapi: 3.1.0
+info:
+  title: Common components
+  version: 1.0.0
+
+components:
+  responses:
+    BadRequest:
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              error:
+                type: string
+                example: Bad request. Please provide valid parameters.
+    Unauthorized:
```

Comment:
This file will be removed in a rebase once the branch I copied it from is merged.